docs/doxygen/platforms/msw_install.md

```diff
@@ -381,54 +375,54 @@ depending on the compiler used.
 
 The full list of the build settings follows:
 
-BUILD=release
+* BUILD=release <br/>
```

Comment:
Maybe use sub-subsections instead of an itemized list and avoid the need for the (ugly) `<br/>`?